src/ripple/protocol/STArray.h

```diff
@@ -71,11 +71,35 @@ class STArray final
         return emplace(n, buf, std::move(*this));
     }
 
-    void push_back (const STObject & object)
+    STObject& operator[] (std::size_t j)
+    {
+        return v_[j];
+    }
+
+    STObject const& operator[] (std::size_t j) const
+    {
+        return v_[j];
+    }
+
+    STObject& back() { return v_.back(); }
```

Comment:
Would it be worth adding a `STObject const& back() const { return v_.back(); }` as well, or is this function only used along with `push_back`, which would make a `const` version pointless?